src/interpolators.jl

```diff
@@ -80,6 +81,7 @@ function delaunay_mesh(positions::AbstractVector{<: Point{2}})
     m = delaunay(convert(Matrix{Float64}, hcat(first.(positions), last.(positions))))
     return GeometryBasics.Mesh(Makie.to_vertices(m.points), Makie.to_triangles(m.simplices))
 end
+=#
```

Comment:
I would do line comments for each impacted line instead of a block comment -- it makes it much more obvious when looking at a given line that it's been commented out

Reply:
I used block comments cause that is what was used to comment out the badly behaving MNE interpolation, so I used the same reasoning when commenting out the badly behaved Delaunay interpolation. But I changed that now to line commenting.